docs/generateDocs.ps1

```diff
@@ -101,12 +101,20 @@ Write-Host "Repo root $repo_root, build root $build_root"
 # Clear output dir
 if( $incremental -eq $false ) {
     Write-Host "Clear previous version from $build_root"
-    Remove-Item -Force -Recurse $build_root
+    Remove-Item -Force -Recurse $build_root -ErrorAction Ignore
 }
 
 mkdir -ErrorAction Ignore $build_root | out-null
 mkdir -ErrorAction Ignore $cache_root | out-null
 
+if ((Get-ChildItem $build_root | Measure-Object).Count -ne 0)
+{
+    $err = "Cannot clean $build_root folder"
+    Write-Error $err
+    Write-Host "##vso[task.complete result=Failed;]$err"
```

Comment:
Nice. For future me https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash